src/pymovements/utils/archives.py

```diff
@@ -149,18 +157,37 @@ def _extract_tar(
         Path to the directory the file will be extracted to.
     compression: str | None
         Compression filename suffix.
+    resume: bool
+        Resume if archive was already previous extracted.
+    verbose: int
```

Comment:
as it's basically just a boolean switch I would use `bool` instead of `int`. otherwise there's potential confusion about the effect of the verbosity level.

we can still "upgrade" to `int` in case we have different verbosity levels, as positive `int` is "backwards compatible" to `bool`.

Reply:
we could bool(verbose) it I guess